content/yaml-publishing/google-cloud-storage.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Google Cloud Storage
+description: How to publish build artifacts to Google Cloud Storage using codemagic.yaml
+weight: 14
+---
+
+In order to publish your generated artifacts to Google Cloud Storage:
+
+1. Log in to your Google Cloud platform [console](https://console.cloud.google.com/). Create a new service account under **IAM & Admin > Service Accounts**.Find more information about how to create a service account [here](https://docs.codemagic.io/knowledge-base/google-services-authentication/#creating-a-service-account). Give **Storage Object Admin** access to the newly created service account which needs to be followed by generating a key as JSON for the service account. This will cause a JSON key file to be downloaded. As the last step in the process, you should create an environment variable called **GCLOUD_STORAGE_KEY** and save the contents of the Service Account JSON to the the variable. More information about environment variable groups can be found [here](https://docs.codemagic.io/variables/environment-variable-groups/). 
```

Comment:
```suggestion
1. Log in to your Google Cloud platform [console](https://console.cloud.google.com/).
 
2. Create a new service account under **IAM & Admin > Service Accounts** and give **Storage Object Admin** access to the newly created service account. Find more information about how to create a service account [here](https://docs.codemagic.io/knowledge-base/google-services-authentication/#creating-a-service-account).

3. Generate a key as JSON for the service account. This will cause a JSON key file to be downloaded. 

4. In the Codemagic web app you should create an environment variable called **GCLOUD_STORAGE_KEY** and add the contents of the Service Account JSON as its value. More information about environment variable groups can be found [here](https://docs.codemagic.io/variables/environment-variable-groups/). 
```